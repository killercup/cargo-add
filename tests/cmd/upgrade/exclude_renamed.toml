bin.name = "cargo-upgrade"
args = ["upgrade", "--exclude", "rx"]
status = "success"
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
