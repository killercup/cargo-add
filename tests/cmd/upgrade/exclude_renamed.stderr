    Upgrading toml_edit v0.1.5 -> v99999.0.0
